src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/CreateNewUserWithRolesTest.java

```diff
@@ -64,7 +64,7 @@ public void returns_404_when_organisation_identifier_not_found() {
         Map<String, Object> newUserResponse =
                 professionalReferenceDataClient.addUserToOrganisation("AB83N5K", userCreationRequest);
 
-        assertThat(newUserResponse.get("http_status")).isEqualTo("404");
+        assertThat(newUserResponse.get("http_status").toString().startsWith("4"));//TODO be more specific

```

Comment:
Can u please add proper assert in the above test. why y removed 404 to 4?

Reply:
I did this b/c it was Shreedhar's responsibility to resolve the error code logic which seems to have been implemented incorrectly in a few instance. But I will address this now for the sake of expediency